### ISSUE TITLE
[docs] Getting started could be easier

### ISSUE THREAD

**Issue by gerdneuman** [shipshapecode/tether#166]
I came over here from this [stackoverflow comment](http://stackoverflow.com/questions/6794000/fixed-position-but-relative-to-container#comment49310309_6794000). I really like tether but getting started was really hard. In the end it took me about 8 hours for what was in the end 6 lines of code. 

I thought to give some feedback on what I found difficult and what would have helped me:
- The documentation does not say which files of the repo are necessary. I learnt from #142 that one only needs the js file and nothing else. I hope, I can send PR later on this to update at least the README.md. 
- There are two places of documentation: Something (e.g. installation) is only in this repo's [README.md](https://github.com/HubSpot/tether#tether), something else (demos) only at http://github.hubspot.com/tether/. Some things are twice and overlap (usage). I'd prefer one single place for the documentation, maybe just have http://github.hubspot.com/tether/ and point to it from the README.md
- Same as above for the examples: Some are shown inline within http://github.hubspot.com/tether/, some linked at another page at http://github.hubspot.com/tether/examples/list_of_examples/
- This page is hard to find, because _all_ links on the left side at http://github.hubspot.com/tether/ are broken
- While looking at the examples or demos I found it hard to understand how these where done (CSS, JS, HTML markup). It would be helpful if one could fiddle around with each them. I would have loved if the demos and examples would be hosted at codepen or jsfiddle. This way the examples/demos would be very easy to investigate and also to adapt and play around with. 

Bottom line: tether is great but could sell better ;)

**Comment by gerdneuman** [comment]
Jeez, there's even a 3rd documentation page ;) http://tether.io/docs/welcome/ IIRC I saw this at the very beginning (it is only linked in the header of the README.md), lost it, but now accidentically found it again...

**Comment by TrevorBurnham** [comment]
Yes, these docs are definitely very stale. Any help is appreciated.

**Comment by rwwagner90** [comment]
We are going to work on documentation via JSDoc in the coming months.
